vcd/datasource_vcd_vapp_vm_test.go

```diff
@@ -41,7 +41,8 @@ func TestAccVcdVappVmDS(t *testing.T) {
 					resource.TestCheckResourceAttr(resourceName, "name", "web1"),
 					resource.TestCheckResourceAttr(resourceName, "storage_profile", "*"),
 					resource.TestCheckResourceAttrSet(resourceName, "href"),
-					resource.TestCheckResourceAttrSet(resourceName, "description"),
+					resource.TestCheckResourceAttr(resourceName, "description", ""),
```

Comment:
This was a bug previously. The code wasn't setting any description and expecting it to be non empty (inherited from template by default)